book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './rental'
+
+class Book
+  def initalize(title, author)
+    @title = title
+    @author = author
+  end
+
+  attr_accessor :title, :author
+
+  has_many :book
+end
```

Comment:
- [ ] You required the rental class in the book class correctly. After that, you should create `add_rental` method to be invoked upon rental creations for a book record. For example:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```